ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -6,6 +6,11 @@ def matrix_rank(a, tol=None, valiate_args=False, name=None):
     return ivy.matrix_rank(a, tol)
 
 
+def adjoint(matrix, name=None):
+    matrix = ivy.Array.matrix_transpose(matrix)
```

Comment:
I found a bug here, you should not use Array.matrix_transpose like this as it is not a static method and should only be used by a class object, or this error happens  AttributeError: 'ResourceVariable' object has no attribute '_data'. 
Instead, you should use this ivy.matrix_transpose function directly like: matrix = ivy.matrix_transpose(matrix)